cmake/FindPackageHandleStandardArgs.cmake

```diff
@@ -0,0 +1,605 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
Why did you add this, this is part of your `CMake` installation...

Reply:
Same as above. Will do more testing and see what needs fixing.